applications/GeoMechanicsApplication/tests/cpp_tests/test_newmark_Pw_scheme.cpp

```diff
@@ -125,18 +129,23 @@ KRATOS_TEST_CASE_IN_SUITE(NewmarkPwSchemeUpdate_SetsDtPressure, KratosGeoMechani
 KRATOS_TEST_CASE_IN_SUITE(InitializeNewmarkPwScheme_SetsTimeFactors, KratosGeoMechanicsFastSuite)
 {
     constexpr double theta = 0.75;
+
     NewmarkQuasistaticPwScheme<SparseSpaceType, LocalSpaceType> scheme(theta);
 
-    Model model;
-    ModelPart& model_part = CreateValidModelPart(model);
-    constexpr double delta_time = 3.0;
+    Model            model;
+    ModelPart&       model_part             = CreateValidModelPart(model);
+    constexpr double delta_time             = 3.0;
     model_part.GetProcessInfo()[DELTA_TIME] = delta_time;
 
+    CompressedMatrix A;
+    Vector           Dx;
+    Vector           b;
     scheme.Initialize(model_part);
```

Comment:
This initialize might be redundant here if I'm correct

Reply:
No, this Initialize initializes many things and put true to mIsInitialised, which is checked in the next line.